src/dftbp/dftb/densitymatrix.F90

```diff
@@ -446,4 +455,277 @@ end subroutine makeDensityMtxCplxBlacs
 
 #:endif
 
+
+#:if WITH_MAGMA
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!!!!!  MAGMA GPU routines    !!!!!                                                   
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> Make a regular density matrix for the real wave-function case in the GPU
+  !> Note: In order to save memory, the eigenvectors (which should be intent(in) parameters) are
+  !> overwritten and then restored again
+
```

Comment:
Doxygen might have problems with the blank line between the comment and the associated routine
```suggestion
```